src/main/java/tuteez/model/person/lesson/LessonManager.java

```diff
@@ -0,0 +1,84 @@
+package tuteez.model.person.lesson;
+
+import java.util.HashMap;
+import java.util.TreeSet;
+
+/**
+ * A container for all Lesson instances
+ * Provides:
+ *  1. Methods to add and delete
+ *  2. Check for clashing lessons
+ */
+public class LessonManager {
+    private final HashMap<Day, TreeSet<Lesson>> dayLessonsMap = new HashMap<>(7);
+
+    /**
+     * Constructs a new {@code lessonManager} and initializes an empty schedule
```

Comment:
Method is very clear with good javadoc comments that made it easy to understand!